### PR TITLE
Transmogrifier

### PR DESCRIPTION
**Pull request by cdw9**
Updates for Python 3

### CODE REVIEW

File: transmogrifier/import.rst [hunk 1]
```diff
@@ -43,11 +43,11 @@ In your ploneconf.migration package, there are two ways provided to import the d
 This is where you'll want to start.
 No code needs to be written for this, it's already available in the migration package.
 
-* Go to the Management Interface > portal_quickinstaller,
-  install ``mysite.migration (default)``, if it is not already installed
-* Then go to portal_setup > Import tab
-* In the first dropdown, find ``ploneconf.migration (import)``
+* Go to ZMI > portal_setup > Import tab,
```
Comment (stevepiercy) [inline transmogrifier/import.rst]:
```
Go to :menuselection:`ZMI > portal_setup > Import` tab,
```

File: transmogrifier/import.rst [hunk 2]
```diff
@@ -43,11 +43,11 @@ In your ploneconf.migration package, there are two ways provided to import the d
...
+* Find ``ploneconf.migration (default)`` in the dropdown
 * Click 'Import all steps'
```
Comment (stevepiercy) [inline transmogrifier/import.rst]: s/b either guilabel or menuselection role, whichever is appropriate.

File: transmogrifier/import.rst [hunk 3]
```diff
@@ -43,11 +43,11 @@ In your ploneconf.migration package, there are two ways provided to import the d
...
+* Then select ``ploneconf.migration (import)`` in the dropdown,
```
Comment (stevepiercy) [inline transmogrifier/import.rst]: s/b either guilabel or menuselection role, whichever is appropriate.

File: transmogrifier/import.rst [hunk 4]
```diff
@@ -43,11 +43,11 @@ In your ploneconf.migration package, there are two ways provided to import the d
...
+  and click 'Import all steps'
```
Comment (stevepiercy) [inline transmogrifier/import.rst]: s/b either guilabel or menuselection role, whichever is appropriate.

File: transmogrifier/import.rst [hunk 5]
```diff
@@ -141,9 +141,13 @@ Here are some tips for debugging those errors.
    and is sometimes easier than working with the debugger to find the actual problem.
    Right before the traceback, you will see the output from the ``logger`` for the item being imported.
    You can grep through the entire export folder for the path output by the logger.
-   Tip: If you find a problem in the data, don't change the data in the export!
+
+.. tip::
+
+   If you find a problem in the data, don't change the data in the export!
    Unless you know for sure that you will not be exporting the data again,
    it's best to handle the issue in the import code.
```
Comment (stevepiercy) [inline transmogrifier/import.rst]: Above should be indented to match the indent of item 3 above, else it is outside the list and in a separate context.

File: transmogrifier/pipeline.rst [hunk 6]
```diff
@@ -207,6 +209,4 @@ This will take the ``language`` key from the item dictionary,
 and change the ``value`` to whatever we set,
 in this case it will be the string ``'en-us'``.
 Your value may be different, depending on what language you set as your site language.
-Check the Languages control panel in Site Setup to see how it is set.
-
-Next: `Blueprints <blueprints>`
\ No newline at end of file
+Check the Languages control panel in Site Setup to see what value you should use.
```
Comment (stevepiercy) [inline transmogrifier/pipeline.rst]: s/b either guilabel or menuselection role, whichever is appropriate.

**Comment by cdw9** [comment]
Thanks for the detailed review, Steve! I will work on getting these fixes in

File: transmogrifier/basics.rst [hunk 7]
```diff
@@ -61,18 +61,29 @@ The created package can be used as an add-on in an existing buildout,
...
    $ cd ploneconf.migration
-   $ virtualenv env --python=python2.7
+   $ virtualenv env --python=python3.7
    $ env/bin/pip install zc.buildout
    $ env/bin/buildout
    $ bin/instance fg
 
 This will start up the instance in foreground mode for you,
 and will be accessible in your browser at http://localhost:8080.
 Click the 'Create a new Plone site' button, and create a site with the id ``Plone``.
```
[the remark this answers is not in the document]
Comment (cdw9) [reply transmogrifier/basics.rst]: that's fancy, I didn't know about that one 👍

**Pull request merged**